contractsV5/bridges/bancor/KyberBancorReserve.sol

```diff
@@ -0,0 +1,230 @@
+pragma solidity 0.5.11;
+
+import "../../IERC20.sol";
+import "../../IKyberReserve.sol";
+import "../../WithdrawableV5.sol";
+import "../../UtilsV5.sol";
+import "./mock/IBancorNetwork.sol";
+
+contract KyberBancorReserve is IKyberReserve, Withdrawable, Utils {
+
+    uint constant internal BPS = 10000; // 10^4
+
+    address public kyberNetwork;
+    bool public tradeEnabled;
+    uint public feeBps;
+
+    IBancorNetwork public bancorNetwork; // 0x0e936B11c2e7b601055e58c7E32417187aF4de4a
+
+    IERC20 public bancorEth; // 0xc0829421C1d260BD3cB3E0F06cfE2D52db2cE315
+    IERC20 public bancorToken; // 0x1F573D6Fb3F13d689FF844B4cE37794d79a7FF1C
+
+    constructor(
+        address _bancorNetwork,
+        address _kyberNetwork,
+        uint _feeBps,
+        address _bancorEth,
+        address _bancorToken,
+        address _admin
+    )
+        public
+    {
+        require(_bancorNetwork != address(0), "constructor: bancorNetwork address is missing");
+        require(_kyberNetwork != address(0), "constructor: kyberNetwork address is missing");
+        require(_bancorEth != address(0), "constructor: bancorEth address is missing");
+        require(_bancorToken != address(0), "constructor: bancorToken address is missing");
+        require(_admin != address(0), "constructor: admin address is missing");
+        require(_feeBps < BPS, "constructor: fee is too big");
+
+        bancorNetwork = IBancorNetwork(_bancorNetwork);
+        bancorToken = IERC20(_bancorToken);
+        bancorEth = IERC20(_bancorEth);
+
+        kyberNetwork = _kyberNetwork;
+        feeBps = _feeBps;
+        admin = _admin;
+        tradeEnabled = true;
+
+        require(bancorToken.approve(address(bancorNetwork), 2 ** 255));
```

Comment:
set decimals missing.

Reply:
Since we will hardcode the decimals, set decimals won't be needed here